handler.go

```diff
@@ -145,6 +145,16 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 		return
 	}
 
+	// If authentication is enabled and there are no users yet, make sure
+	// the first statement is creating a new cluster admin.
+	if h.AuthenticationEnabled && h.server.UsersLen() == 0 {
```

Comment:
I discussed this style issue with @benbjohnson  -- at what point do we push variables into `New` constructors, versus just exposing them. Ben felt that unless a variable is actually needed during construction, it was cleaner Go code to just expose it as a public attribute. Otherwise constructor params lists can get tediously long. I feel his pain.